### DISCUSSION TITLE
Add input prompt inside the integrated terminal

### DISCUSSION THREAD

**Discussion by jamestut** [microsoft/vscode-discussions#202] [Extension Development QnA] [answered]
Hi there, is it possible to add an input prompt on each of the integrated terminal? I have attached a simple illustration on what I am trying to achieve.

<img width="1136" alt="Screen Shot 2022-10-28 at 9 12 02 am" src="https://user-images.githubusercontent.com/15940448/198408173-cca68fa8-8e99-427a-af25-fe16d6dc6926.png">

**Comment by digitarald** [comment, marked as answer]
There is currently no API for that. I suggest filing a feature request with some arguments on what you are trying to achieve. As the terminals are based on other terminal designs, maybe there is prior art that helps to make a case.

https://github.com/microsoft/vscode/issues/new/choose
